src/JBCCIPSucker.sol

```diff
@@ -66,7 +99,8 @@ contract JBCCIPSucker is JBSucker, ModifiedReceiver {
     /// @param remoteToken Information about the remote token being bridged to.
     function _sendRoot(uint256 transportPayment, address token, JBRemoteToken memory remoteToken) internal override {
         bool localIsNative = token == JBConstants.NATIVE_TOKEN;
-        bool willSendWeth = localIsNative && block.chainid == 1;
+        // TODO: change to only check chainid 1 for prod
```

Comment:
should we pass these values to-be-changed in the constructor as well? so we can just instantiate with different values?

Reply:
definitely.